### ISSUE TITLE
compatability for v5

### ISSUE THREAD

**Issue by chrismccoy** [MickeyKay/better-font-awesome-library#14]
any plans on adding font awesome 5?

**Comment by MickeyKay** [comment]
Yes! I finally got around to this. Thanks for your patience. Please see https://github.com/MickeyKay/better-font-awesome-library/pull/23. I’d love your review if you have bandwidth.
